svgnative/example/testD2D/TestD2D.cpp

```diff
@@ -12,116 +12,190 @@ governing permissions and limitations under the License.
 
 #define WIN32_LEAN_AND_MEAN
 
-#include <tchar.h>
 #include <windows.h>
-#include <unknwn.h>
-#include <D2d1.h>
-#include <memory>
+#include <d2d1.h>
+#pragma comment(lib, "d2d1")
 
+#include "basewin.h"
 #include "svgnative/SVGDocument.h"
 #include "svgnative/ports/d2d/D2DSVGRenderer.h"
 
-/* We need to include the Gdiplus lib */
-#pragma comment (lib, "D2d1.lib")
+template <class T> void SafeRelease(T** ppT)
+{
+	if (*ppT)
```

Comment:
The rest of the code uses spaces as indentation (actually, I thought we have code style checks in place but maybe the bots are only checking PRs on main.)